### PR TITLE
Update of release notes

### PR DESCRIPTION
**Pull request by johannes-b**
Signed-off-by: Johannes <johannes.braeuer@dynatrace.com>

### CODE REVIEW

File: releasenotes/releasenotes_develop.md [hunk 1]
```diff
@@ -1,53 +1,87 @@
...
+
+*Gain visibility into the integrations connected to Keptn* - Keptn 0.8.4 starts to implement the Uniform mockup that has been released/presented with Keptn 0.8.1. This new Bridge screen brings insights into the Keptn-services (aka Integrations) that are connected to a Keptn control-plane, allows troubleshooting by retrieving their error logs, and enables creating/deleting secrets for integrations.
 
-Keptn 0.8.x
 ---
 
 **Key announcements:**
 
-:tada: 
+:tada: *Troubleshooting support for Integration*: To support troubleshooting integrations without connecting to the environment that runs them, errors are sent to Keptn and displayed in the Uniform screen of a project.
```
Comment (christian-kreuzberger-dtx) [inline releasenotes/releasenotes_develop.md:9]: Troubleshooting support for Integration**s** (add an s?)

File: releasenotes/releasenotes_develop.md [hunk 2]
```diff
@@ -1,53 +1,87 @@
...
 
-:star: 
+:star: *Creating/Deleting Secrets for Integrations*: To not rely on the Keptn CLI to manage a secret for integrations, the Bridge allows creating/deleting secrets. This is supported for integrations that are running on a Keptn control-plane since the public Keptn API does yet not allow querying secrets.
```
Comment (christian-kreuzberger-dtx) [inline releasenotes/releasenotes_develop.md:11]: Suggestion: To not rely on the Keptn CLI to manage ~a~ secret**s** for integrations, ~the~ Bridge allows creating/deleting secrets.

File: releasenotes/releasenotes_develop.md [hunk 3]
```diff
@@ -1,53 +1,87 @@
...
 **Key announcements:**
 
-:tada: 
+:tada: *Troubleshooting support for Integration*: To support troubleshooting integrations without connecting to the environment that runs them, errors are sent to Keptn and displayed in the Uniform screen of a project.
 
-:star: 
+:star: *Creating/Deleting Secrets for Integrations*: To not rely on the Keptn CLI to manage a secret for integrations, the Bridge allows creating/deleting secrets. This is supported for integrations that are running on a Keptn control-plane since the public Keptn API does yet not allow querying secrets.  
 
-:rocket: 
+:rocket: *Customization of Bridge*: With this release, the Keptn Bridge can get a custom *look-and-feel* by providing a custom logo, title, and/or stylesheet. More details on this feature are available [here](https://github.com/keptn/keptn/tree/0.8.4/bridge#custom-look-and-feel).
```
Comment (christian-kreuzberger-dtx) [inline releasenotes/releasenotes_develop.md:13]: Also same here, are we always having a *the* in front of Keptn Bridge? I think it reads better just saying "With this release, Keptn Bridge can ..."

File: releasenotes/releasenotes_develop.md [hunk 4]
```diff
@@ -1,53 +1,87 @@
...
+
+- *distributor*:
+  - Forward log messages of execution plane Keptn-services to Keptn core [4030](https://github.com/keptn/keptn/issues/4030)
+  - Send data of subscribed Keptn-services (via distributors) to uniform [4031](https://github.com/keptn/keptn/issues/4031)
+
+- *helm-service*: 
+  - *Fixed*: Helm-service lost its resource requests/limits [4250](https://github.com/keptn/keptn/issues/4250)
+
+- *lighthouse-service*:
+  - *Fixed*: "Response time degradation in lighthouse-service" when spamming get-sil-events [4065](https://github.com/keptn/keptn/issues/4065)
```
Comment (christian-kreuzberger-dtx) [inline releasenotes/releasenotes_develop.md:75]: Typo: *get-sil-events* -> *get-sli-events*

File: releasenotes/releasenotes_develop.md (hunk 1 above)
Comment (johannes-b) [reply releasenotes/releasenotes_develop.md:9]: thx - fixed

File: releasenotes/releasenotes_develop.md (hunk 2 above)
Comment (johannes-b) [reply releasenotes/releasenotes_develop.md:11]: valid point. In German, we tend to use the article, but not needed in English. 

Thanks, for the hint - I will review with an eye on that.

**Review by christian-kreuzberger-dtx — APPROVED** [review]
LGTM

**Review by bacherfl — APPROVED** [review]

**Pull request merged**